lib/registerHelpers.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const camelCase = require('lodash.camelcase')
+const flow = require('lodash.flow')
+const kebabCase = require('lodash.kebabcase')
+const mixin = require('lodash.mixin')
+const snakeCase = require('lodash.snakecase')
+const upperFirst = require('lodash.upperfirst')
+
+// best solution for now https://github.com/lodash/lodash/pull/942
+const m = mixin({'pascalCase': flow(camelCase, upperFirst)})
+
+module.exports = handlebars => {
+  handlebars.registerHelper({
```

Comment:
I got some internal feedback about the casing of these helpers. I see inconsistent use across the Handlebars documentation, so any direction from the community is helpful here.